styles.css

```diff
@@ -0,0 +1,502 @@
+  display: flex;
+  text-align: end;
+}
+
+.container-fluid {
+  background-image: url('./images/illustration-header-mobile.png');
+  background-position: right 0 bottom -43px;
+  background-repeat: no-repeat;
+  background-size: 135px;
+  color: #fff;
+  padding: 0;
+  background-color: #1c1a19;
+  align-items: flex-start;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+}
+
+.element {
+  display: flex;
+  color: dark;
+  list-style: none;
+  gap: 1.5rem;
+  position: absolute;
+  padding-top: 1.2rem;
+}
+
+.home {
+  color: white;
+  display: flex;
+  flex: column;
+  justify-content: center;
+}
+
+.heading-home {
+  color: #ff6b00;
+  margin-top: 1.2rem;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.card-text {
+  position: relative;
+  width: 327px;
+  height: 123px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.card-title {
+  color: white;
+  width: 327px;
+  height: 27px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work-heading {
+  margin-top: 640px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 83px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+.img-fluid {
+  width: 327px;
+  height: 237px;
+  margin-left: 24px;
+  margin-right: 24px;
+  column-gap: 80px;
+  margin-top: 104px;
+}
+
+.img-fluid2 {
+  width: 327px;
+  height: 237px;
+  margin-left: 190px;
+  margin-right: 24px;
+  column-gap: 80px;
+  margin-top: 4px;
+}
+
+h1::after {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  position: absolute;
+  display: block;
+}
+
+.heading {
+  margin-top: 84px;
+  margin-left: 16px;
+  width: 295px;
+  height: 81px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.work-card-text-1 {
+  margin-top: 12px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 147px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.work {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 1.2rem;
+  flex-direction: column;
+  align-items: baseline;
+}
+
+.skills {
+  display: flex;
+  list-style: none;
+  margin-left: -44px;
+  margin-right: 24px;
+}
+
+.badge {
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 49px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  margin-top: 12px;
+  margin-left: 16px;
+  margin-bottom: 22px;
+}
```

Comment:
[OPTIONAL]

- [ ] I suggest that you remove the fixed `width` and `height` since you already have `padding` set up which is good, and also I suggest that you remove the margins you set and instead add `margin: auto` this works better in this case, it would make your badges look like the screenshot below, which is similar to the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). 😁

![image](https://user-images.githubusercontent.com/20567503/176990507-f2b872b2-e732-4e68-bc14-70d25a1cb69c.png)